dep/libmpq/CMakeLists.txt

```diff
@@ -24,9 +24,9 @@ file(GLOB sources libmpq/*.c libmpq/*.h)
 add_library(mpq STATIC ${sources})
 set_target_properties(mpq PROPERTIES LINKER_LANGUAGE CXX)
 
-if(WIN32)
+# if(WIN32)
```

Comment:
It looks like this is also required on a mac.

Maybe we could have a `/mac` and `/linux` folders for the final change, or just use the same file for all of the platforms.

I guess this depends on if there is platform specific config.

Reply:
This is fixed now